lib/rocket_job/jobs/upload_file_job.rb

```diff
@@ -91,7 +91,7 @@ def job_implements_upload
       end
 
       def file_exists
-        return if upload_file_name.nil? || File.exist?(upload_file_name)
+        return if upload_file_name.nil? || File.exist?(upload_file_name) || upload_file_name =~ /^s3:/
```

Comment:
Instead of checking for s3 explicitly could we parse the file name using `URI` and then check if the uri scheme is empty or `file` before performing the file existence check?

Reply:
For an example see: https://github.com/rocketjob/iostreams/blob/master/lib/io_streams/s3.rb#L8

Reply:
👍

Reply:
okay updated.  lmk. 😄